pkgs/development/libraries/zlib/default.nix

```diff
@@ -23,14 +23,14 @@ assert splitStaticOutput -> static;
 
 stdenv.mkDerivation (rec {
   pname = "zlib";
-  version = "1.2.12";
+  version = "1.2.13";
 
   src = fetchurl {
     urls =
       [ "https://www.zlib.net/fossils/zlib-${version}.tar.gz"  # stable archive path
```

Comment:
Arch fetches from https://github.com/archlinux/svntogit-packages/blob/2dfdf85b4155a2c94b01518d9de55c7263d92fe9/trunk/PKGBUILD#L13

Reply:
That's bad, as that URL disappears on every release.  They only keep the last one there.

Reply:
It's why the "stable archive path" comment is there.